client/my-sites/sharing/buttons/preview-action.jsx

```diff
@@ -1,40 +1,41 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { omit, noop } from 'lodash';
 
-module.exports = React.createClass( {
+export default React.createClass( {
```

Comment:
Do we have to use `React.createClass` instead of using ES6 classes for the React component here? They are preferred.